sway-lib-std/src/inputs.sw

```diff
@@ -82,7 +109,21 @@ pub fn input_pointer(index: u64) -> u64 {
     }
 }
 
+/// Get amount field from input at `index`.
+/// If the input's type is `InputCoin` or `InputMessage`,
+/// return the amount as an Option::Some(u64).
+/// Otherwise, returns Option::None.
```

Comment:
```suggestion
/// return the amount as an `Option::Some(u64)`.
/// Otherwise, returns `Option::None`.
```

Reply:
Would be nice to standardize this across the whole file by adding the backticks like this.